CHANGES.md

```diff
@@ -1,3 +1,33 @@
+# grmtools 0.13.0 (XXXX-XX-XX)
+
+## Breaking changes
+
+* `cfgrammar::yacc::grammar::YaccGrammarSpan` now returns `Option<Span>` rather
+  than `Option<&Span>`.
```

Comment:
I believe this should be:
`cfgrammar::yacc::grammar::YaccGrammar::token_span`,

That function had not made it to a release yet, unfortunately I didn't get the signature right first commit.

So it should perhaps be:
`cfgrammar::yacc::grammar::YaccGrammar::token_span`, was added.

Reply:
Fixed in 193e052.